internal/translator/utils/templates.go

```diff
@@ -20,7 +20,13 @@ func ReplaceValues(templateString string, words []linguakit.Word) string {
 		vars[fmt.Sprintf("Word%d", i+1)] = word
 	}
 
-	parsedTemplate, error := template.New("tmp").Option("missingkey=zero").Parse(templateString)
+	//parsedTemplate, error := template.New("tmp").Option("missingkey=zero").Parse(templateString)
+	parsedTemplate, error := template.New("tmp").Funcs(template.FuncMap{
+		"StartWithVowel":     StartWithVowel,
+		"StartWithConsonant": StartWithConsonant,
+		"FirstLetter":        FirstLetter,
```

Comment:
As I understand the code, look like this returns the first letter of a string. So a better name could be `GetFirstLetter`